src/stock-items/add-stock-item/packaging-units/packaging-units.component.tsx

```diff
@@ -110,6 +112,28 @@ const PackagingUnitRow: React.FC<{
     resolver: zodResolver(packageUnitSchema),
   });
 
+  const handleDelete = (e) => {
+    e.preventDefault();
+    deleteStockItemPackagingUnit(row.uuid).then(
+      () => {
+        showToast({
+          critical: true,
+          title: `Delete packing item `,
+          kind: "success",
+          description: `Stock Item packing unit deleted Successfully`,
+        });
+      },
+      (error) => {
+        showNotification({
+          title: `Error Deleting a stock item packing unit`,
+          kind: "error",
```

Comment:
Same here translation of texts will be nice to have. This will come in handy for other implementers